src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -309,7 +325,7 @@ public BitbucketRepository getRepository() throws IOException {
      */
     @Override
     public void postCommitComment(@NonNull String hash, @NonNull String comment) throws IOException {
-        postRequest(String.format(API_COMMIT_COMMENT_PATH, getUserCentricOwner(), repositoryName, hash), new NameValuePair[]{ new NameValuePair("text", comment) });
+        postRequest(String.format(API_COMMIT_COMMENT_PATH, getUserCentricOwner(), repositoryName, hash), Arrays.asList(new BasicNameValuePair("text", comment)));
```

Comment:
You can use Collections.singletonList instead and avoid importing Arrays

Reply:
done